<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>139</width>
    <height>54</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FF7tk Widget Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QComboBox" name="combo_widget">
      <item>
       <property name="text">
        <string>Select Widget...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OptionsWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Materia Editor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ItemList</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CharEditor</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MetaData Creator</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SlotSelect</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PhsListWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MenuListWidget</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lgp Utility</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LocationViewer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ChocoboManager</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AchievementEditor</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="sb_scale">
      <property name="prefix">
       <string>Widget Scale : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="item_list_box">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>Editable Item Combo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="itemList_box">
         <property name="title">
          <string>GroupBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="materia_editor_box">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>setStarSize(int)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QSpinBox" name="sb_materia_editor_setStarSize">
            <property name="maximum">
             <number>128</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QCheckBox" name="cbEditableMateriaCombos">
               <property name="text">
                <string>Editable Combo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_materia_editor_setEditable">
               <property name="text">
                <string>is Editable</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="editor_box">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="char_editor_box">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Initilze Char Data</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="cb_charEditor">
              <property name="text">
               <string>is Editable</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Debug</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Editable Combos </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="charEditor_box">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="phsListBox">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="menuListBox">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="dialog_preview_box">
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="metadata_box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_showmetaData">
         <property name="text">
          <string>Show Dialog</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="slotSelect_Box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbShowLoad">
         <property name="text">
          <string>Allow Loading of new file from dialog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_slotSelect">
         <property name="text">
          <string>Select File to Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="lgp_Box">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>View / Extract</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btn_lgpSelect">
            <property name="text">
             <string>Select Lgp To Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupLgpView">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="lblLgpCompany">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblLgpProduct">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listLgpFile"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnExtractLgp">
               <property name="text">
                <string>Extract To...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Create / Add File(s)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Demo Coming Soon</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="locListBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Advanced Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="innerLocBox">
         <property name="title">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="ChocoboManagerBox"/>
    </item>
    <item>
     <widget class="QGroupBox" name="AchievementEditor_Box">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QPushButton" name="btn_loadAchievement">
         <property name="text">
          <string>Select Achievement to Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="achievementEditor_Frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_saveAchievement">
         <property name="text">
          <string>Save File</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
